server/src/instant/storage/sweeper.clj

```diff
@@ -0,0 +1,155 @@
+(ns instant.storage.sweeper
+  (:require
+   [chime.core :as chime-core]
+   [instant.storage.s3 :as instant-s3]
+   [instant.util.s3 :as s3-util]
+   [instant.util.tracer :as tracer]
+   [clojure.tools.logging :as log]
+   [instant.jdbc.aurora :as aurora]
+   [honey.sql :as hsql]
+   [instant.jdbc.sql :as sql]
+   [instant.config :as config]
+   [instant.discord :as discord])
+
+  (:import
+   (java.time Duration Instant)
+   (java.lang AutoCloseable)
+   (java.time.temporal ChronoUnit)))
+
+(defonce max-loops 10) ;; max loops per sweep job
```

Comment:
I would just make these defs, otherwise they won't change when you change the value in the code and re-load the file.

`defonce` is more for things where redefining it would break something, e.g. an atom where you're storing some state.